README.md

```diff
@@ -1103,6 +1103,70 @@ build --tool_java_runtime_version=nixpkgs_java
 
 ##### Bazel 6
 
+#### with with [Bzlmod](https://bazel.build/versions/6.5.0/external/overview#bzlmod)
+
+Add the following to your `MODULE.bazel` file to depend on `rules_nixpkgs`, `rules_nixpkgs_java`, and nixpgks:
+```bzl
+bazel_dep(name = "rules_nixpkgs_core", version = "0.12.0")
+bazel_dep(name = "rules_nixpkgs_java", version = "0.12.0")
+bazel_dep(name = "rules_java", version = "7.3.1")
+bazel_dep(name = "platforms", version = "0.0.9")
```

Comment:
these will eventually not be necessary anymore according to https://github.com/tweag/rules_nixpkgs/blob/master/testing/java/MODULE.bazel#L17-L20 . but i cannot leave a `TODO` comment here.